libmpv/qt_opengl/main.cpp

```diff
@@ -6,7 +6,8 @@ int main(int argc, char *argv[])
     QApplication a(argc, argv);
     // Qt sets the locale in the QApplication constructor, but libmpv requires
     // the LC_NUMERIC category to be set to "C", so change it back.
-    setlocale(LC_NUMERIC, "C");
+    //setlocale(LC_NUMERIC, "C");
+    // Commented because it's not compiling on macOS
```

Comment:
Use `#ifndef Q_OS_MAC` so that other platforms will work as expected.  Don't comment out the correct way of running libmpv.